### DISCUSSION TITLE
Consider applying for support from OSGEO or OGC?

### DISCUSSION THREAD

**Discussion by Mec-iS** [georust/geo#1173] [Ideas]
Hi, I am a geospatial engineer and I follow this industry since ten years now, beside being a contributor to Rust-based projects.

As geospatial package are most of the time mission critical and/or highly scientifically scrutinized, I was wondering if the leading contributors are considering getting in touch with major geospatial institutions to receive support for the long-term sustainability of this project. Some like me would like to have Rust in the geospatial pipelines but the lack of official "endorsement" is a big challenge to overcome.

For example I see OSGEO participating at Google Summer Of Code, it would be great to have this crate been contributed to with an organized effort. Also, another example, it is possible to apply for [OSGEO UK funding](https://uk.osgeo.org/fundingguidelines.html) that will allow more experts to contribute to geospatial Rust crates. I guess there are other initiatives like this to support the development effort.

I can consider helping in facilitating these processes if you think it is the way to go. Let me know.

**Comment by urschrei** [comment]
It would be wonderful to have some support, but in some ways it's a chicken-and-egg problem: the core contributors all (?) have full-time jobs, so funding at the scale in question isn't immediately useful to us (please chime in if you want to implement geometry buffering for money). What we **do** need are more motivated contributors, and people who can help us to build momentum by driving adoption in the companies they work in, then contributing features back and helping to maintain the project's many (many, many) crates.

If someone would like to pay a motivated intern to work on new features or certification work as part of e.g. GSoC, we would welcome that. I will however note that I harbour severe misgivings about collaborating with the OGC. Things like [this](https://www.ogc.org/requests/ogc-requests-public-comment-on-features-and-geometries-json-part-1-core-standard/) are only the tip of the iceberg – as an entity, it seems wholly focused on entrenching the position of its members, with the production of high-quality geospatial software often a secondary concern.

**Reply by Mec-iS** [reply]
> If someone would like to pay a motivated intern to work on new features or certification work as part of e.g. GSoC

That's what I meant, GSOC (Google) pays stipends to students for the 12 weeks programme during summer so they are highly motivated. That's a great way to attract talents and try later to keep them interested. Plus there is a selection process in which the really interested ones submit a project proposal to be accepted by the organization. I have been admin for a GSOC organization and been mentoring for some time in the programme, I can help with the organizational bits.
